clang-tools-extra/test/clang-tidy/checkers/Inputs/DDLNHeaders/project1/dir1/tmp.cc

```diff
@@ -0,0 +1,11 @@
+#include <project1/dir1/cls1.hh>
+#include <project1/dir1/header1.hh>
+#include <project1/dir1/header2.hh>
+#include <project1/dir1/header2.hh>
+#include <project1/dir2/header3.hh>
+#include <project2/header3.hh>
+//
+//
+// C1HECK-MESSAGES: :[[@LINE-8]]:10: warning: <project1/dir1/cls1.hh> and <project1/dir1/header1.hh> should be in different groups [daedalean-include-order]
```

Comment:
C1HECK? (How) does this actually work?

Shouldn't there alsobe other messages expected about dir1 and dir2 headers in different groups, and project1/project2?